### ISSUE TITLE
Menu and Bag (Pokemon ORAS)

### ISSUE THREAD

**Issue by josserumfle** [weihuoya/citra#776]
In the previous version of Citra, it was possible to make Pokémon Omega Ruby and Alpha Sapphire run stably at 60FPS but there is a problem related to the bag, and that is that when you open it, the FPS drastically drops to more than half, and that makes That equipping and using objects is quite complicated, the same happens with the contests tab in the Pokémon team section, where you can see their contest statistics.

I hope my feedback helps the development somewhat.

Citra current version: 316086c6a

![Screenshot_2023-04-20-19-56-15-874-edit_com miui securitycenter](https://user-images.githubusercontent.com/15163411/233510292-0d89bd5c-8c7e-4351-8fbc-6a56d4886c3f.jpg)

**Comment by Tappy-tech** [comment]
60 FPS? Sounds like you used a 60 FPS cheat. I'd advise you to not use it for Pokémon games. Some others may have a different outcome, though. Like, instead of just Fast Forwarding, it uncaps.

**Comment by josserumfle** [comment]
Interestingly, no, the 60FPS cheat actually worsens the game's performance, what I meant is that after the penultimate update, it is now possible to play Pokémon on Citra, which was not possible before because the emulator was not yet available optimized for gaming.

Ultra sun runs great on the emulator just by changing a small option in the configuration menu, but in ORAS it's different.

**Comment by Gelo021** [comment]
ar u perhaps using custom texture ?

**Comment by josserumfle** [comment]
> ar u perhaps using custom texture ?

No, I don't use any custom texture, I run the game as it comes, with the only change that I have the 1.4 update installed for Pokémon ORAS, all my games are up to date to the latest version, If necessary, I could make a video showing the current performance of the game.

**Comment by Samuel180161** [comment]
HelloI'm having the same problems, did you manage to resolve it somehow?

**Comment by AndresNorm** [comment]
Did you try Clearing the Shader Cache after the Update?

**Comment by Zeshax** [comment]
> Did you try Clearing the Shader Cache after the Update?

I'm going to try what you told me and I'll update again as soon as I've tried it.
